source/Camera/Advanced/CaptureHDRLoop/CaptureHDRLoop.cpp

```diff
@@ -18,11 +18,13 @@ int main()
         std::cout << "Connecting to camera" << std::endl;
         auto camera = zivid.connectCamera();
 
+        const auto cameraModel = camera.info().modelName().toString().substr(0, 9);
```

Comment:
So is this based on us being so lucky that both "zivid one" and "zivid two" are strings of length 9? 
Seems brittle. Consider using camera.info().revision().major() instead to distinguish cameras.

Reply:
We discussed this earlier on teams, and we found that for customers revisions are confusing and should be avoided. 
But yes this is based on the string length being 9. 
Seems like we want to create an enum to easy seperate cameras.

Reply:
Are there cases where the strings get other lengths? 
As long as 0-9 is "Zivid xxx" it will work, right?

Reply:
If this has already been discussed then I don't need to stop it.

Yeah what I meant is that there is no guarantee that all future model names will fit in nine characters. "Zivid Three" is the obvious example. Of course this is far in the future, so I was making a general point about robust and nice code.

Reply:
I agree that the public CameraInfo should have had an enum :)

Reply:
I will make sure to push on camera model as enum to make it robust for future cameras :) 
I concider this comment as resolved then